src/thunderbots/software/test/primitive/catch_primitive.cpp

```diff
@@ -9,12 +9,10 @@
 
 TEST(CatchPrimTest, constuct_with_no_params_test)
 {
-    const std::string catch_prim_name = "Catch Primitive";
-
     CatchPrimitive catch_prim = CatchPrimitive(0, 0, 0, 0);
 
-    EXPECT_EQ(int(), catch_prim.getRobotId());
-    EXPECT_EQ(catch_prim_name, catch_prim.getPrimitiveName());
+    EXPECT_EQ(0, catch_prim.getRobotId());
+    EXPECT_EQ(CatchPrimitive::PRIMITIVE_NAME, catch_prim.getPrimitiveName());
```

Comment:
I'm not a huge fan of this, just because I could change `CatchPrimitive::PRIMITIVE_NAME` to "wooglyboogly" and this would pass. This applies to all instances of this below. Ideally I think we just have both checks? (ie. the new and old version)?

Reply:
Having both seems redundant. I'll put this back to raw strings.

Reply:
I replaced the `create_primitive_with_no_parameters` tests with `primitive_name_tests` that test against the raw strings. The `create_primitive_with_no_parameters` tests were redundant and now we can still use the `ClassName::INTENT_NAME` in the rest of the tests.